docs/src/content/docs/hi/guides/sidebar.mdx

```diff
@@ -0,0 +1,541 @@
+
+## डिफ़ॉल्ट साइडबार
+
+डिफ़ॉल्ट रूप से, Starlight स्वचालित रूप से आपके दस्तावेज़ की संचिका प्रणाली संरचना के आधार पर साइडबार प्रविष्टि के रूप में प्रत्येक फ़ाइल की `title` गुण का उपयोग करके एक साइडबार उत्पन्न करेगा।
+
+उदाहरण के लिए, निम्नलिखित फ़ाइल संरचना दी गई है:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+      - reference/
+        - configuration.md
+
+</FileTree>
+
+निम्नलिखित साइडबार स्वचालित रूप से उत्पन्न हो जाएगा:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'guides',
+			items: [
+				{ label: 'अवयवों', link: '/guides/components/' },
+				{ label: 'अंतर्राष्ट्रीयकरण (i18n)', link: '/guides/i18n/' },
+			],
+		},
+		{
+			label: 'reference',
+			items: [
+				{ label: 'कॉन्फ़िगरेशन संदर्भ', link: '/reference/configuration/' },
+			],
+		},
+	]}
+/>
+
+[स्वतः-निर्मित समूह](#स्वतः-निर्मित-समूह) अनुभाग में स्वतः-निर्मित साइडबार के बारे में और जानें।
+
+## लिंक और लिंक समूह जोड़ें
+
+अपने साइडबार [लिंक](#लिंक) और [लिंक के समूहों](#समूह) (एक सिमटने वाले शीर्ष लेख के भीतर) को कॉन्फ़िगर करने के लिए, `astro.config.mjs` में [`starlight.sidebar`](/hi/reference/configuration/#sidebar) गुण का उपयोग करें।
+
+लिंक और समूहों को मिलाकर, आप विभिन्न प्रकार के साइडबार लेआउट बना सकते हैं।
+
+### लिंक
+
+`label` और `link` गुणों वाले ऑब्जेक्ट का उपयोग करके किसी आंतरिक या बाहरी पृष्ठ पर एक लिंक जोड़ें।
+
+```js "label:" "link:"
+starlight({
+	sidebar: [
+		// CSS और स्टाइलिंग मार्गदर्शिका का लिंक।
+		{ label: 'CSS और स्टाइलिंग', link: '/guides/css-and-tailwind/' },
+		// Astro वेबसाइट का एक बाहरी लिंक।
+		{ label: 'Astro', link: 'https://astro.build/' },
+	],
+});
+```
+
+उपरोक्त कॉन्फ़िगरेशन निम्नलिखित साइडबार उत्पन्न करता है:
+
+<SidebarPreview
+	config={[
+		{ label: 'CSS और स्टाइलिंग', link: '/guides/css-and-tailwind/' },
+		{ label: 'Astro', link: 'https://astro.build/' },
+	]}
+/>
+
+### समूह
+
+आप एक सिमटने वाले शीर्षक के अंतर्गत संबंधित लिंक को एक साथ समूहित करके अपने साइडबार में संरचना जोड़ सकते हैं।
+समूहों में लिंक और अन्य उप-समूह दोनों हो सकते हैं।
+
+`label` और `items` गुणों वाले ऑब्जेक्ट का उपयोग करके एक समूह जोड़ें।
+`label` का उपयोग समूह के शीर्षक के रूप में किया जाएगा।
+`items` सरणी में लिंक या उपसमूह जोड़ें।
+
+```js /^\s*(label:|items:)/
+starlight({
+	sidebar: [
+		// "मार्गदर्शिका" लेबल वाले लिंक का एक समूह।
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{ label: 'अवयवों', link: '/guides/components/' },
+				{ label: 'अंतर्राष्ट्रीयकरण (i18n)', link: '/guides/i18n/' },
+				// लिंक का एक नेस्टेड समूह।
+				{
+					label: 'स्टाइलिंग',
+					items: [
+						{ label: 'CSS', link: '/guides/css-and-tailwind/' },
+						{ label: 'Tailwind', link: '/guides/css-and-tailwind/' },
+						{ label: 'Shiki', link: '/guides/css-and-tailwind/' },
```

Comment:
```suggestion
				{ label: 'अवयवों', link: '/hi/guides/components/' },
				{ label: 'अंतर्राष्ट्रीयकरण (i18n)', link: '/hi/guides/i18n/' },
				// लिंक का एक नेस्टेड समूह।
				{
					label: 'स्टाइलिंग',
					items: [
						{ label: 'CSS', link: '/hi/guides/css-and-tailwind/' },
						{ label: 'Tailwind', link: '/hi/guides/css-and-tailwind/' },
						{ label: 'Shiki', link: '/hi/guides/css-and-tailwind/' },
```

Prefixing the link for the same reason, e.g. this [example](https://starlight.astro.build/ko/guides/sidebar/#%EA%B7%B8%EB%A3%B9).